insights/parsers/systemd_analyze.py

```diff
@@ -0,0 +1,74 @@
+"""
+SystemdAnalyzeBlame - command ``systemd-analyze blame``
+=======================================================
+
+This module parses the output of command ``systemd-analyze blame``.
+"""
+from insights.specs import Specs
+from insights import CommandParser, parser
+from insights.parsers import keyword_search
+
+
+@parser(Specs.systemd_analyze_blame)
+class SystemdAnalyzeBlame(CommandParser):
+    """Parse the output of ``systemd-analyze blame``.
+
+    Typical output::
+
+        33.080s cloud-init-local.service
+        32.423s unbound-anchor.service
+         2.773s kdump.service
+         1.699s dnf-makecache.service
+         1.304s cloud-init.service
+         1.073s initrd-switch-root.service
+          939ms cloud-config.service
+          872ms tuned.service
+          770ms cloud-final.service
+
+    Examples:
+
+        >>> 'cloud-init-local.service' in output
+        True
+        >>> output.search('cloud-init-local.service') == {'time': 33.08, 'unit_name': 'cloud-init-local.service'}
+        True
+        >>> output.search('cloud-init.service').get('time')
+        1.304
+        >>> output.search('foobar.service')
+        >>> 'foobar.service' in output
+        False
+
+    Attributes:
+        data (list): List of key-value pair for each Systemd Unit.
+                     The time to initialize is converted to Seconds.
+                     Ex: ``[{'time': 0.08, 'unit_name': 'auditd.service'}]``
+        units (list): Unit names from the output.
+    """
+    def parse_content(self, content):
+        self.data = []
```

Comment:
Instead of using `.data` here, why not base the parser class on `dict` as in `class SystemdAnalyzeBlame(CommandParser, dict)`.  This would provide some additional features and avoids use of the generic attribute `data`.  See [podman_inspect](https://github.com/RedHatInsights/insights-core/blob/2ddf017d9237a5ae0f3783d15803431965d1b976/insights/parsers/podman_inspect.py#L17) as an example.